pkgs/servers/dynomite/default.nix

```diff
@@ -0,0 +1,25 @@
+{ fetchFromGitHub, stdenv, autoreconfHook, libressl, git }:
+stdenv.mkDerivation rec {
+  pname = "dynomite";
+  version = "0.6.14";
+
+  src = fetchFromGitHub {
+    owner = "Netflix";
+    repo = "dynomite";
+    rev = "67166e3cdc194caa16843c613d2bd9a0e86e86d3";
+    sha256 = "1lk6hrkad4zpfjvmn5hskzfr748zp7ccwqbhy46skw5hvsy2fa68";
+  };
+
+  buildInputs = [ autoreconfHook git libressl ];
+
+  NIX_CFLAGS_COMPILE = [
+    "-Wformat"
```

Comment:
From the nixpkgs manual:

> The following flags are enabled by default and might require disabling with hardeningDisable if the program to package is incompatible.
format

>    Adds the -Wformat -Wformat-security -Werror=format-security compiler options. At present, this warns about calls to printf and scanf functions where the format string is not a string literal and there are no format arguments, as in printf(foo);. This may be a security hole if the format string came from untrusted input and contains %n.

Is this actually required? Quickly skimming the manual tells me this might already be included.

Reply:
This also doesn't seem resolved

Reply:
As far as I read the docks those flags are set when using the cc-wrapper but otherwise you have to set them your self